src/OSPSuite.UI/UIConstants.cs

```diff
@@ -12,7 +12,6 @@ public static class Colors
       {
          // These colors change with the skin, so we use a method to read the latest up-to-date value from the skin
          public static Color Disabled => skinColorFor("ReadOnly");
-         public static Color TextDisabled => skinColorFor("DisabledText");
```

Comment:
Removing this - it makes sense because 'DisabledText' we know reduces the contrast as an indicator. Now we are going to change the background of cells, but leave the foreground to default - that should retain most of the contrast for both dark and light themes.